### DISCUSSION TITLE
Is there a way to set up a superuser to manage the registed users?

### DISCUSSION THREAD

**Discussion by BinGuoGuo** [exceptionless/Exceptionless#1061] [Ideas]
Is possible to provide a page to allow register and manager the registed users?
And group the users by role, and grant each role a different permissions?
And allow anonymous users to check some public pages?

**Comment by niemyjski** [comment]
We currently don't have fine grain roles, but you can already manage a list of users. What set of pages would you want people to access anonymously?

**Reply by BinGuoGuo** [reply]
1. We can only manage users in an existing organizations, but not users registered in the entire system（there is no root user）.

2. About pages that can be accessed anonymously，When an exception occurs in the project, I want to push the exception information to our IM APP through hooks, and notify those who subscribed to this event, but I don't want them to have to register an account first, then join the organization, and then log in to the account to view details.

I noticed that Jenkins has a similar page to access build status anonymously（but I didn't assess security，just an idea）:)
